config/swagger.yaml

```diff
@@ -2006,6 +2006,33 @@ paths:
           description: Invalid contract
           schema:
             $ref: '#/definitions/Error'
+  /contract/decode-data:
```

Comment:
Worth a line in release notes? Though optional.

Reply:
Done